conf/init/data_migration.py

```diff
@@ -18,7 +22,7 @@ def current_migration():
 
 
 def main():
-    print(current_migration())
+    logger.info(current_migration())
```

Comment:
This will break container startup; this output is piped into the alembic command